dll/win32/imm32/imm.c

```diff
@@ -105,6 +111,57 @@ Imm32NotifyAction(HIMC hIMC, HWND hwnd, DWORD dwAction, DWORD_PTR dwIndex, DWORD
     return TRUE;
 }
 
+HKL WINAPI ImmLoadLayout(DWORD dwLayout, PIMEINFOEX pImeInfoEx)
```

Comment:
Shouldn't the first parameter be a HKL too?

Reply:
Made it `HKL`. OK.